packages/tools-standard/package.json

```
@@ -13,14 +13,15 @@
   },
   "version": "1.0.0",
```

Lets bump the version, since this is now pure ESM.

```suggestion
  "version": "2.0.0",
```